### PR TITLE
clarify webp image format support on safari

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

similar changes as #29353

### CODE REVIEW

**Review by pepelsbey — CHANGES_REQUESTED** [review]
Good catch! I’d go a bit further here though. This note sounds a bit complicated. The main point here is that image codecs support in Safari depends on OS version, not the browser version. It’s less relevant for iOS/iPadOS since Safari version strictly depends on OS version, but it becomes more obvious on macOS.

**Comment by jasonren0403** [comment]
> Good catch! I’d go a bit further here though. This note sounds a bit complicated. The main point here is that image codecs support in Safari depends on OS version, not the browser version. It’s less relevant for iOS/iPadOS since Safari version strictly depends on OS version, but it becomes more obvious on macOS.

I'm not quite familar with this field(and also poor English for documenting this), could you directly suggest what should be written here 🤔

File: files/en-us/web/media/formats/image_types/index.md [hunk 1]
```diff
@@ -1363,7 +1363,7 @@ Provide a fallback in either [JPEG](#jpeg_joint_photographic_experts_group_image
   </tbody>
 </table>
 
-> **Note:** Despite having [announced support](https://developer.apple.com/videos/play/wwdc2020/10663/?time=1174) for WebP in Safari 14, as of version 14.0 .webp images do not display natively on a macOS desktop, whereas Safari on iOS 14 does display .webp images properly.
+> **Note:** Despite Apple [announcing support for WebP in Safari 14](https://developer.apple.com/videos/play/wwdc2020/10663/?time=1174), Safari versions earlier than 16.0 don't display `.webp` images successfully on macOS desktop versions earlier than 11/Big Sur. Safari for iOS 14 _does_ display `.webp` images successfully.
```
Comment (teoli2003) [inline files/en-us/web/media/formats/image_types/index.md:1366]: I went ahead and crafted a potential note here:
```suggestion
> **Note:** On Safari for MacOS, WebP support depends on both Safari and MacOS versions. You need Safari 14 or later as well as MacOS Big Sur (11) or a more recent version.
```

Comment (teoli2003) [reply files/en-us/web/media/formats/image_types/index.md:1366]: @jasonren0403, have a look (and eventually commit the suggestion), then re-request review by @pepelsbey.

Comment (pepelsbey) [reply files/en-us/web/media/formats/image_types/index.md:1366]: @teoli2003 officially, it’s macOS, starting with lowercase “m”.

Comment (teoli2003) [reply files/en-us/web/media/formats/image_types/index.md:1366]: Fixed. (I hate brands that doesn't start with a capital, but…)

Comment (jasonren0403) [reply files/en-us/web/media/formats/image_types/index.md:1366]: seems good 🤔

**Review by pepelsbey — APPROVED** [review]
LGTM! Thank you :)

**Pull request merged**